src/cms/models/languages/language_tree_node.py

```diff
@@ -149,12 +195,46 @@ def insert_at(
         allow_existing_pk=False,
         refresh_target=True,
     ):
+        """
+        This function inserts parameters
+
+        :param target: the current target
+        :type target: str
+
+        :param position: the current position
+        :type position: str
+
+        :param save: returns whether or not should be saved
+        :type save: bool
+
+        :param allow_existing_pk: returns whether or not the existing primary key should be allowed
+        :type allow_existing_pk: bool
+
+        :param refresh_target: returns whether or not
+        :type refresh_target: bool
+
+        :return: The inserted node
+        :rtype: ~cms.models.languages.language_tree_node.LanguageTreeNode
+
+        """
         return super().insert_at(
             target, position, save, allow_existing_pk, refresh_target
         )
 
     # pylint: disable=useless-super-delegation
     def move_to(self, target, position="first-child"):
+        """
+        This function moves a node to a target at a defined position
+
+        :param target: the current target
+        :type target: str
+
+        :param position: the current position
+        :type position: str
+
+        :return: The moved node
+        :rtype: ~cms.models.languages.language_tree_node.LanguageTreeNode
+        """
         return super().move_to(target, position)
 
```

Comment:
```suggestion
```
Same justification like above, and please also remove the lines
```
def insert_at(
        self,
        target,
        position="first-child",
        save=False,
```
which I cannot suggest changes for. Thanks!

Reply:
this is done in 7f6b735a48fe6df23b9849e56a232aee64f6e5ef
but i still have to fix a commit message problem...

Reply:
What problem do you mean?

Reply:
I already fixed it, it was about the co-authored statement in commit messages